Formula/mailutils.rb

```diff
@@ -5,6 +5,7 @@ class Mailutils < Formula
   mirror "https://ftpmirror.gnu.org/mailutils/mailutils-3.15.tar.gz"
   sha256 "91c221eb989e576ca78df05f69bf900dd029da222efb631cb86c6895a2b5a0dd"
   license "GPL-3.0-or-later"
+  revision 1
```

Comment:
Can't merge without breaking the `mailutils` bottle on Monterey ARM64 unfortunately.